modules/cnv-configuring-vmi-live-migration-cli.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// cnv_users_guide/cnv-configuring-vmi-eviction-strategy.adoc
+
+[id="cnv-configuring-vmi-live-migration-cli_{context}"]
+= Configuring custom virtual machine instances with the `LiveMigration` eviction strategy 
+
+You only need to configure the `LiveMigration` eviction strategy on custom 
+virtual machine instances. +
```

Comment:
~+~

Reply:
Wanted to put that second sentence on a new line but not new para, but looking at it again it adds nothing. 
Removed the '+' entirely.